cmd/server-main.go

```diff
@@ -36,7 +36,7 @@ var serverFlags = []cli.Flag{
 	cli.StringFlag{
 		Name:  "address",
 		Value: ":9000",
-		Usage: `Bind to a specific IP:PORT. Defaults to ":9000".`,
+		Usage: "Bind to a specific ADDRESS:PORT.",
```

Comment:
Can be:
Usage: "Bind to a specific ADDRESS:PORT. (ADDRESS can be IP or Hostname)"